tests/sentinel/tests/03-runtime-reconf.tcl

```diff
@@ -1 +1,53 @@
 # Test runtime reconfiguration command SENTINEL SET.
+source "../tests/includes/init-tests.tcl"
+set num_sentinels [llength $::sentinel_instances]
+
+
+
+test "Set parameters in normal case" {
+
+    set info [S 0 SENTINEL master mymaster]
+    set origin_quorum [lindex $info 35]
```

Comment:
Avoid accessing by `lindex`. It is less readable and can break easily. 
Instead you can write something like:
```
    set info [S 0 SENTINEL master mymaster]
    set origin_quorum [dict get $update_info quorum]
```

Reply:
Thanks, Got it. I will use it in later PR.